telegram/ext/_jobqueue.py

```diff
@@ -626,7 +650,7 @@ def jobs(self) -> Tuple["Job[CCT]", ...]:
             Tuple[:class:`Job`]: Tuple of all *scheduled* jobs.
         """
         return tuple(
-            Job._from_aps_job(job)  # pylint: disable=protected-access
+            Job.from_aps_job(job)  # pylint: disable=protected-access
```

Comment:
```suggestion
            Job.from_aps_job(job)
```
We're not accessing a protected method anymore?